CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
What happens if teams are not using docker? Does installation of uav_ros_simulation by cloning the github repo include larics/storm_gazebo_ros_magnet or we need to update instructions for installing without docker?